chain-signatures/node/src/protocol/signature.rs

```diff
@@ -258,10 +258,13 @@ impl SignQueue {
                 .with_label_values(&[indexed.chain.as_str(), my_account_id.as_str()])
                 .inc();
 
-            let request = self.organize_request(stable, participants, indexed, 0);
+            let mut request = self.organize_request(stable, participants, indexed, 0);
+            if request.indexed.timestamp_sign_queue.is_none() {
```

Comment:
Why is this field optional in the first place? Can we make it non optional and ensure timestamp is added at the indexer level?